src/assets/scripts/client/trafficGenerator/spawnPatternReadme.md

```diff
@@ -0,0 +1,80 @@
+## Spawn Patterns
```

Comment:
@erikquinn Really need your eyes on this one.  I feel like I'm missing something and there could certainly be more detail here.

This may not be this files final place in the app, but it's a good spot for now.  We can always link to it.